IndustryConnect_Week_Microservices/Models/IndustryConnectWeek2Context.cs

```diff
@@ -19,15 +19,21 @@ public IndustryConnectWeek2Context(DbContextOptions<IndustryConnectWeek2Context>
 
     public virtual DbSet<CustomerSale> CustomerSales { get; set; }
 
+    public virtual DbSet<CustomerSaleView> CustomerSaleViews { get; set; }
+
+    public virtual DbSet<CustomerSaleView1> CustomerSaleView1s { get; set; }
+
     public virtual DbSet<Product> Products { get; set; }
 
     public virtual DbSet<Sale> Sales { get; set; }
 
     public virtual DbSet<Store> Stores { get; set; }
 
+    public virtual DbSet<Store1> Store1s { get; set; }
```

Comment:
Why do you need to set up another Store in here?

Reply:
It is created for some extra practice for queries in week2.
but now i don't need that table so i will remove it.